src/pages/_app.tsx

```diff
@@ -26,18 +27,20 @@ export default function App({ Component, pageProps }) {
   const currentProduct = pageProps.product || null
 
   return (
-    <ErrorBoundary FallbackComponent={Error}>
-      <DeviceSizeProvider>
-        <CurrentProductProvider currentProduct={currentProduct}>
-          <Layout
-            {...pageProps?.layoutProps}
-            openConsentManager={openConsentManager}
-          >
-            <Component {...pageProps} />
-          </Layout>
-        </CurrentProductProvider>
-      </DeviceSizeProvider>
-      <ConsentManager />
-    </ErrorBoundary>
+    <SSRProvider>
```

Comment:
Do we want this to wrap everything?

Reply:
Based on [the docs](https://react-spectrum.adobe.com/react-aria/SSRProvider.html), just a +1 that this looks right to me!